site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -2,370 +2,107 @@
-alt="Options page link in the UI", width="800", height="299" %}
-  <figcaption>
-    Link to the Options page.
-  </figcaption>
-</figure>
-
-{% endColumn %}
-
-{% Column %}
-
-<figure>
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/BM11QeGCThsUNTlsZbAe.png", 
-alt="Context Menu Options page", width="357", height="222" %}
+The manifest is the configuration file of a Chrome extension. It is a required JSON file that must be located at the root of the project. It provides the browser with a blueprint of the extension, with important information such as:
 
-  <figcaption>
-    Options page in the extension's context menu.
-  </figcaption>
-</figure>
+- The name of the extension, a description of what it does, the current version number, and what icons to use.
+- The [Chrome APIs][api-ref] keys and [permissions][doc-perms] that the extension needs.
+- The files assigned as the extension service worker, the popup HTML file, the options page, the content scripts, etc.
 
-{% endColumn %}
+The [Manifest keys][doc-manifest] article contains the complete list of default and optional keys. For copy-paste-ready code samples, check out the [Manifest examples][doc-manifest-examples].
 
-{% endColumns %}
+### The service worker {: #background_script }
 
-See [Give users options][docs-options] to learn more.
+The extension service worker is the event manager of a Chrome extension. It is a script that runs in the background within the context of the browser. It monitors and responds to events in the browser. For example, when a new tab is created, a new bookmark is added, the extension is installed, the extension toolbar icon is clicked, etc.
 
-### Additional HTML files {: #html-files}
+It can access all the [Chrome APIs][api-ref], but as a service worker, it can't use the DOM APIs that the global window object provides. It also cannot modify the page's content.
 
-An extension can also have other HTML files that are not declared in the manifest. All extension HTML files can access the [Chrome APIs][section-apis] and can use script tags including Javascript files, but cannot declare inline JavaScript.
+See [Handling events in the extension service worker][doc-sw] for more details. 
 
-You can open these pages using the web api [window.open()][mdn-window-open], the Chrome APIs
-[windows.create()][api-window-create], or [tabs.create()][api-create-tab].
+### Content scripts {: #content-scripts }
 
-## Extension files {: #files }
+Extensions use content scripts to inject code into host pages. They allow the extension to interact with and modify the pages in the browser. For example, they can insert a new element on the page, change the style of a website, modify the [DOM][mdn-dom] elements, etc. 
 
-### Referencing extension files {: #ref-files }
+Content Scripts share the same origin as their host page and they have access to a limited number of [Chrome APIs][api-ref]. See [Understanding content scripts][doc-content-scripts] for more details.
```

Comment:
> Content Scripts share the same origin as their host page

While technically true (calling `new URL(window.location.href).origin` in a content script and directly in the page will give you the same value), I don't think this is the best thing to highlight here. Non-web developers likely won't have a sense of what this means or it's implications, and I expect web developers will mostly think of origins as the boundary for cookies or localStorage.

I think it would be better to focus this line on isolated worlds. 

```suggestion
Content Scripts share access to the same DOM tree as the host page but run in a separate JavaScript environment. They also have access to a limited number of [Chrome APIs][api-ref]. See [Understanding content scripts][doc-content-scripts] for more details.
```